examples/scripts/compute_reference_Morris_indices.py

```
@@ -1,38 +1,95 @@
 
+# %%
 first_order_string = get_string(result.first_order_indices)
 print("First order indices")
 print(first_order_string)
 total_order_string = get_string(result.total_order_indices)
 print("Total order indices")
 print(total_order_string)
+
+
+# %%
+print(f"Compare reference to computed indices")
```

Ruff (0.15.15)

[error] f-string without any placeholders

Remove extraneous `f` prefix

(F541)